sagemaker/launch/launch_infer.py

```diff
@@ -16,16 +16,20 @@
     Launch SageMaker inference task
 """
 import os
-import sagemaker
 import argparse
-import boto3 # pylint: disable=import-error
-
-from graphstorm.config import SUPPORTED_TASKS
 
-from sagemaker.pytorch.estimator import PyTorch
+import boto3 # pylint: disable=import-error
 import sagemaker
+from sagemaker.pytorch.estimator import PyTorch
 
 INSTANCE_TYPE = "ml.g4dn.12xlarge"
+SUPPORTED_TASKS = {
```

Comment:
Is it possible to add a unit test to ensure SUPPORTED_TASKS is synced with graphstorm.config.SUPPORTED_TASKS ?

Reply:
I'd be hesitant to add that because these scripts are supposed to be live outside the library, so we'd need to import them in some hacky way. 

I'd argue it's very unlikely this list will change often and if it does the SageMaker regression tests should catch that.

Reply:
That makes sense.